nx-X11/programs/Xserver/composite/compalloc.c

```diff
@@ -481,14 +633,21 @@ compAllocPixmap (WindowPtr pWin)
 	DamageRegister (&pWin->drawable, cw->damage);
 	cw->damageRegistered = TRUE;
     }
+
+    /* Make sure our borderClip is up to date */
+    REGION_UNINIT(pWin->drawable.pScreen, &cw->borderClip);
+    REGION_COPY(pWin->drawable.pScreen, &cw->borderClip, &pWin->borderClip);
```

Comment:
Taken care of in https://github.com/ArcticaProject/nx-libs/pull/125/commits/d64eabf3d4b233bdbc10f40fa3df653d489101fc - I must have overlooked these when rebasing against 3.6.x. Most other places lead to a merge commit and were ported to `Region*` appropriately.